sdk/pom.xml

```diff
@@ -316,8 +316,7 @@
 					<include>new_mc_blocks/mc_block_empty_sidechains</include>
 					<include>new_mc_blocks/mc_block_create_3_sidechains</include>
 					<include>new_mc_blocks/mc_block_forward_transfer_2_sidechains</include>
-					<include>new_mc_blocks/mc_block_forward_transfer_3_backward_transfer</include>
```

Comment:
What was inside `mc_block_forward_transfer_2_sidechains` and `mc_block_forward_transfer_3_backward_transfer`?
What case was tested? Do we still have that test cases somewhere else?